packages/css-engine/src/core/to-value.ts

```diff
@@ -1,44 +1,50 @@
 import type { StyleValue } from "@webstudio-is/css-data";
 import { DEFAULT_FONT_FALLBACK, SYSTEM_FONTS } from "@webstudio-is/fonts";
 
-type ToCssOptions = {
-  withFallback: boolean;
-};
-
-const defaultOptions = {
-  withFallback: true,
-};
+export type TransformValue = (styleValue: StyleValue) => undefined | StyleValue;
 
 // exhaustive check, should never happen in runtime as ts would give error
 const assertUnreachable = (_arg: never, errorMessage: string) => {
   throw new Error(errorMessage);
 };
 
+const fallbackTransform: TransformValue = (styleValue) => {
```

Comment:
do you want to move the transform to the application in a separate step?

Reply:
because if you moved it right away, instead of providing a transform for withFallback: false, you would be able to turn this around and provide transform always and not provide it for withFallback: false

Reply:
This is kinda default stuff, not builder specific. Do we need to move it into application?

Reply:
> because if you moved it right away, instead of providing a transform for withFallback: false, you would be able to turn this around and provide transform always and not provide it for withFallback: false

I will need to put it into every addStyleRule in builder and in sdk. And somehow share it caz the logic is quite complicated

Reply:
> This is kinda default stuff, not builder specific. Do we need to move it into application?

yes because this transform contains application level logic and dependency

Reply:
Alright, someday then 😄

Reply:
we can move it later as well if its too much work

Reply:
I mean yes, better later